pkg/asset/machines/openstack/machines.go

```diff
@@ -111,10 +112,19 @@ func provider(clusterID string, platform *openstack.Platform, mpool *openstack.M
 				Name: fmt.Sprintf("%s-%s", clusterID, role),
 			},
 		},
-		// TODO(flaper87): Trunk support missing. Need to add it back
+		Trunk: trunkSupportBoolean(trunk),
 	}, nil
 }
 
+func trunkSupportBoolean(trunkSupport string) (result bool) {
+	if trunkSupport == "1" {
```

Comment:
why do we need this integer <-> boolean  check? Can't' we just set it to true/false?

Reply:
When the support was added, celebdor was checking that type supported on terraform were string, list and map:
"According to the terraform documentation the valid types are "string, list, and map". There is no example of bool in the repository and digging around I saw that on 0.11 strings are the type supposed to be used for boolean values (when containing "true", "1", "false", "0"). Thankfully, this will change with 0.12."

So, that is how the value is loaded on pkg/asset/installconfig/openstack/openstack.go